server/prisma/schema.prisma

```diff
@@ -13,26 +13,30 @@ datasource db {
 }
 
 model User {
-  id      Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id          Int      @id @default(autoincrement())
+  createdAt   DateTime @default(now())
+  updatedAt   DateTime @updatedAt
   
-  firstName String 
-  lastName String
-  email    String @unique 
-  password String
+  firstName   String 
+  lastName    String
+  email       String @unique 
+  password    String
   // post POST[]
-  profile Profile?
-  role Role @default(STUDENT)
+  profile     Profile?
+  role        Role @default(STUDENT)
 }
 
 model Profile {
-  id     Int    @id @default(autoincrement())
-  age Int 
-  bio    String?
-  contactInfo String
-  user   User   @relation(fields: [userId], references: [id])
-  userId Int    @unique
+  id          Int    @id @default(autoincrement())
+  age         Int? 
+  bio         String?
+  subjects    String?
+  rate        Decimal?
+  phone       String?
+  city        String?
+  state       String?
+  user        User   @relation(fields: [userId], references: [id])
```

Comment:
Might be good idea to have cascade on delete so profile is deleted automatically when user is deleted?

```
user User @relation(fields: [userId], references: [id], onDelete: Cascade)
```

Reply:
I was looking for an easy way to do cascading deletes in the documentation, and couldn't find this. It actually suggested we make the relation optional and then delete the profile when the user is deleted. I will change it to this.